src/components/tasks/mine_card/index.tsx

```diff
@@ -0,0 +1,32 @@
+import { FC } from 'react';
+import classNames from '@/components/tasks/mine_card/card.module.scss';
+import { task } from '@/components/constants/types';
+
+type Props = {
+  content: task,
+};
+
+function informationElement(title: string, value: string) {
+  return (
+    <span className={classNames.statusElement}>
+      <span className={classNames.statusLable}>{`${title}: `}</span>
+      <strong>{value}</strong>
+    </span>
+  );
+}
+const Card: FC<Props> = ({ content }) => {
+  const {
+    title,
+    startedOn,
+    status,
+  } = content;
+  return (
+    <div className={classNames.card}>
+      <span className={classNames.prTitle}>{title}</span>
+      {informationElement('Started', startedOn)}
+      {informationElement('Status', status)}
+    </div>
+  );
+};
+
+export default Card;
```

Comment:
Any specific reason for creating a new card component separate from mine tasks when we already have a card component?
If not, we can just use `tasks/card` instead of creating this new component.

Reply:
I thought Like for mine, we need to see less info but I will change it to card only